README.md

```diff
@@ -1,87 +1,161 @@
+- Tutorial Description - You can change the description of your tutorial _or_ press Enter to accept the Proposal description.
+- Select the keywords you would like to use (maximum of 5) - This list can be scrolled with the arrow keys, but unless you want to add any tags here it will already have the tags you added when submitting the proposal so you can press Enter to continue. 
+- Push changes? - **Answer "y" for Yes**.
 
+When this is complete, a new directory will be created for your tutorial inside the `tutorials` directory, matching the tutorial slug.
 
-## Writing a tutorial
+Inside, you will find:
+- `/content/` - Directory containing the tutorial markdown files
+- `/content/index.mdx` - The main markdown file for your tutorial
+- `/content/assets/` - Place any images used in your tutorial here, linking to images is explained in the tutorial template
+- `/builderdao.config.json` - Tutorial metadata
+- `/builderdao.lock.json` - Configuration lockfile
 
-By running `yarn install` in the root of the project you should have a BuilderDAO CLI available
+## Commit, Push, Create a PR
 
-### Initializing tutorial
+> If this is your first time pushing a commit to a tutorial branch, you will need to set the upstream and `git` will prompt you to do so, otherwise you can just `git push`.
 
-In order to initialize tutorial run
+Once you are finished writing the first draft of your tutorial, save your changes and then commit and push to your fork. Optionally, you can check the files being staged with `git status`: 
 
-```bash
-builderdao tutorial init
+```text
+git status
+git add .
+git commit -m "<YOUR_TUTORIAL_TITLE>"
+git push
 ```
 
-You will be prompted with few questions about your tutorial. After you are done you will have a boilerplate to start writing your tutorial.
-A new folder will be created for your tutorial. Name of the folder is the same as the tutorial slug.
-Inside of this folder you will find:
-- `content` - folder with tutorial markdown files
-- builderdao.config.json - file with tutorial metadata
-- builderdao.lock.json - file with tutorial configuration. THIS FILE SHOULD NOT BE MODIFIED MANUALLY
+> Remember to replace `<YOUR_TUTORIAL_TITLE>` and `<SLUG>` with the correct values for your tutorial. For example, if the title was "HTML 101", the slug would be "html-101".
+
+Now go to your fork of the repo on GitHub (`https://github.com/<YOUR_GITHUB_USERNAME>/kafe`) and [create a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). Make sure that the _base branch_ of your PR is the `dev` branch of the Kafè repo.
+
+## Publishing
 
-### Publishing
+Once your PR is active, the assigned reviewers will look it over carefully and make suggestions for improvement, point out errors and typos, and work with you to ensure a high quality tutorial.
```

Comment:
We should introduce the concept of reviewers before this sentence. Maybe above in the proposal section you can say that, "Reviewers consisting of protocol devrels and community stewards are assigned to your guide to help you take your guide draft to finished product in no time!"

Then here, you can say, "...your assigned reviewers..."